src/lua/LuaInterfaceGame.cpp

```diff
@@ -371,3 +374,12 @@ void LuaInterfaceGame::EventResourceFound(unsigned char player, const MapPoint p
     if(onResourceFound.type() == LUA_TFUNCTION)
         onResourceFound.call<void>(player, pt.x, pt.y, type, quantity);
 }
+
+
+bool LuaInterfaceGame::EventCancelPact(PactType pt, unsigned char canceledByPlayerId, unsigned char targetPlayerId)
```

Comment:
I'd be more verbose: `onPactCancelRequest`. Otherwise it might interfere if we add a real `onCancelPact`. Or we name that other one `onPactCanceled`. But I'd go with both to avoid confusion.

Reply:
Im ok with changing it to onPactCancelRequest.